ports/stm32/boards/ARDUINO_OPTA/pins.csv

```diff
@@ -0,0 +1,95 @@
+# Inputs
+IN_1,PA0_C
+IN_2,PC2_C
+IN_3,PF12
+IN_4,PB0
+IN_5,PF10
+IN_6,PF8
+IN_7,PF6
+IN_8,PF4
+
+# Outputs
+OUT_1,PI6
+OUT_2,PI5
+OUT_3,PI7
+OUT_4,PI4
+RELAY_1,PI6
+RELAY_2,PI5
+RELAY_3,PI7
+RELAY_4,PI4
+
+# Modbus
+UART3_TX,PB10
+UART3_RX,PB11
+UART3_CTS,PB13
+UART3_RTS,PB14
+UART3_RE,PB13
+UART3_DE,PB14
+
+# Expansion port
+UART6_TX,PG14
+UART6_RX,PC7
+I2C3_SDA,PH8
+I2C3_SCL,PH7
+BUS_DETECT,PG8
+
+# Crypto/USB C controller
+I2C1_SDA,PB7
+I2C1_SCL,PB6
+USB_PCON_DETECT,PJ4
+USB_PCON_FAULT,PE7
+USB_PCON_ENABLE,PG1
+
+-USB_DM,PA11
+-USB_DP,PA12
+
+# LEDs
+LED_BUILTIN,PH11
+LED_SYS_1,PH11
+LED_SYS_2,PH12
+LED_SYS_3,PE5
+LEDR,PH11
+LEDG,PH12
+LEDB,PE5
+USER_BUTTON,PE4
+LED_D0,PI0
+LED_D1,PI1
+LED_D2,PI3
+LED_D3,PH15
+LED_RELAY1,PI0
+LED_RELAY2,PI1
+LED_RELAY3,PI3
+LED_RELAY4,PH15
+
+OSCEN,PH1
+-WL_REG_ON,PJ1
+-WL_HOST_WAKE,PJ5
+-WL_SDIO_0,PC8
+-WL_SDIO_1,PC9
+-WL_SDIO_2,PC10
+-WL_SDIO_3,PC11
+-WL_SDIO_CMD,PD2
+-WL_SDIO_CLK,PC12
+-BT_RXD,-PH14
+-BT_TXD,PB9
+-BT_CTS,-PB15
+-BT_RTS,-PA15
+-BT_REG_ON,PJ12
+-BT_HOST_WAKE,PJ13
+-BT_DEV_WAKE,PJ14
+-QSPI2_CS,PG6
+-QSPI2_CLK,PF10
+-QSPI2_D0,PD11
+-QSPI2_D1,PD12
+-QSPI2_D2,PE2
+-QSPI2_D3,PF6
+-ETH_RST,PJ15
+-ETH_RMII_REF_CLK,PA1
+-ETH_MDIO,-PA2
+-ETH_RMII_CRS_DV,-PA7
+-ETH_MDC,-PC1
+-ETH_RMII_RXD0,-PC4
+-ETH_RMII_RXD1,-PC5
+-ETH_RMII_TX_EN,PG11
+-ETH_RMII_TXD0,PG13
+-ETH_RMII_TXD1,PG12
```

Comment:
Please add a newline at the end of the file (not a blank line, a newline on the last line).